src/plugins/terminal/main.js

```diff
@@ -32,10 +32,11 @@ class App {
         this.send({ event: "noop" })
     }
 
+    /** @param {string} input */
     query(input) {
         if (input.startsWith(':')) {
             this.shell_only = true
-            this.last_query = input.substr(1)
+            this.last_query = input.substr(1).trim()
         } else {
             this.shell_only = false
             this.last_query = input.startsWith('t:')
```

Comment:
After 31e92a50d759aa25071ef1474146ac560d651ec0, `sh` commands starting with `:` are trimmed, but terminal commands starting with `t:` are not trimmed:

![Screenshot from 2020-12-11 11-43-20](https://user-images.githubusercontent.com/7199422/101942146-4dc07300-3ba6-11eb-8a92-980ced351431.png)

![Screenshot from 2020-12-11 11-43-14](https://user-images.githubusercontent.com/7199422/101942136-4ac58280-3ba6-11eb-8651-3e2ec3a33599.png)

Should they be consistent?

Reply:
They should